custom/inddex/tests/test_master_report.py

```diff
@@ -97,6 +97,15 @@ def with_db(func):
         yield
 
 
+def setup_module():
+    """Make dividedwerun run all tests in this module on the same node
+
+    This could be eliminated if dividedwerun had a way to determine that
+    tests in this module use a fixture (inddex_domain) that uses the
+    database.
+    """
+
+
```

Comment:
Can you elaborate on how including a `setup_module` function ensures dividewerun runs all tests in this module on the same node?

Reply:
https://github.com/dimagi/commcare-hq/blob/a908d8eb05f6af4285da97dd5ae58998c0a1ad99/corehq/tests/pytest_plugins/dividedwerun.py#L51-L53